src/htm/regions/DatabaseOutRegion.hpp

```diff
@@ -0,0 +1,122 @@
+/* ---------------------------------------------------------------------
+ * HTM Community Edition of NuPIC
+ * Copyright (C) 2013, Numenta, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses.
+ * --------------------------------------------------------------------- */
+
+/** @file
+ * Declarations for DatabaseOutRegion class
+ */
+
+//----------------------------------------------------------------------
+
+#ifndef SRC_HTM_REGIONS_DATABASEOUTREGION_HPP_
+#define SRC_HTM_REGIONS_DATABASEOUTREGION_HPP_
+
+//----------------------------------------------------------------------
+
+#include <htm/engine/RegionImpl.hpp>
+#include <htm/ntypes/Array.hpp>
+#include <htm/types/Types.hpp>
+#include <htm/types/Serializable.hpp>
+#include <htm/ntypes/Value.hpp>
+
+#include <sqlite3.h>
+
+namespace htm {
+
+
+/**
+ *  DatabaseOutRegion is region that takes inputs and writes them to
+ *  the SQLite3 database file.
```

Comment:
Please add a short description what one can do with the sql file, ie "The recorded db file can be visualized using HTMPandaVis at <URL>". 

Also please add a few lines to our Readme section on visualization
https://github.com/htm-community/htm.core#htmpandavis 
About this region, how it's shown in `hello_napi_database` example and that it is intended to be used for visualization with HTMPandaVis,..